### PR TITLE
Update onboarding instructions and README

### PR DESCRIPTION
**Pull request by mrbobbytables**
- Remove list of outdated CNCF funded expenditures
- Updated Steering onboarding instructions
- Document Steering Private Channels
- Document Kubernetes CNCF GB Rep
- Update current CNCF Rep

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/hold 
for review

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by palnabarun — APPROVED** [review]
/lgtm

**Review by soltysh — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -49,44 +49,72 @@ specific committee structure information.
...
-We have two meetings every month.
+The Kubernetes Project is granted one of the two [Developer Representative]
+seats on the [CNCF Governing Board]. This seat may be held by current and
+former Kubernetes Steering Members and is elected to a two year term.
 
-- We hold an open and recorded online meeting where the community is welcome to join the first Monday at 9:30am PT of every month if there is [quorum](charter.md#quorum).
-- We have a closed and not recorded online meeting every 3rd Monday of the month at 9:30am PT if there is [quorum](charter.md#quorum).
+| Name | Profile | Term |
+| ---- | ------- | ---- |
+| Christoph Blecker | **[@cblecker](https://github.com/cblecker)** | 2025 |
```
Comment (pacoxu) [inline README.md:62]: Do we have a Emeritus list here? 😄 Not urgent.
/lgtm

Comment (cpanato) [reply README.md:62]: will be good to have :)

Comment (mrbobbytables) [reply README.md:62]: Added an emeritus section 👍

File: operations/onboarding.md [hunk 2]
```diff
@@ -5,41 +5,31 @@ members and off-board emeritus members.
...
   - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
-  - [ ] [kubernetes/k8s.io] - update steering@kubernetes.io and steering-private@kubernetes.io in `groups/groups.yaml`
-
-- [ ] Add new members to the `steering-private` slack channel and
-  remove emeritus members.
-
+  - [ ] [kubernetes/k8s.io] - update steering@kubernetes.io and
+        steering-private@kubernetes.io in `groups/groups.yaml`
+- [ ] Add new members to the `steering-private`, `steering-gp-rep-private` and,
+     slack channel and `steering-cncf-rep-private` remove emeritus members.
```
Comment (palnabarun) [inline operations/onboarding.md:21]:
```suggestion
     slack channel and `steering-cncf-rep-private`; and removing emeritus members.
```

**Pull request merged**